storyboarding/scripting/compound_commands.rst

```diff
@@ -12,6 +12,405 @@ Loop
+
+.. code-block:: c
+  :linenos:
+  :caption: Character running in Pass-state with transition
+  :name: RunningComplete
+
+  Animation,Pass,Centre,"sb\running\running.png",320,240,6,150,LoopForever
+   F,0,0,,1
+   T,Passing,0,90000
+    F,0,0,,0
+    F,0,1000,,1
+
+The value of 1000 is arbitrary, it has to be set to the actual duration of getting up.
+
+Now the same is done for the `Fail`-layer with the `Failing` trigger. This time it is a sprite, not an animation as Aoba is just lying down.
+
+.. code-block:: c
+  :linenos:
+  :caption: Character lying on the ground
+  :name: Tripped
+
+  Sprite,Fail,Centre,"sb\falling\fallen.png",280,240
+   F,0,0,,1
+   T,Failing,0,90000
+    F,0,0,,0
+    F,0,2000,,1
+
+All that is left is putting the actual animations for falling and getting up in.
+
+.. code-block:: c
+  :linenos:
+  :caption: Character falling
+  :name: Tripping
+
+  Animation,Fail,Centre,"sb\falling\falling.png",280,240,9,150,LoopOnce
+   T,Failing,0,90000
+    F,0,0,,1
+    F,0,1000,,0
+
+.. code-block:: c
+  :linenos:
+  :caption: Character getting up
+  :name: GettingUp
+
+  Animation,Pass,Centre,"sb\gettingup\gettingup.png",320,240,8,150,LoopOnce
+   T,Passing,0,90000
+    F,0,0,,1
+    F,0,2000,,0
+
+Oh hey, that is easy, isn't it?
+
+Yes, too easy to actually work. Otherwise this would be the coolest interactive storyboard in 29 lines ever.
+
+The problem with using animations here is that they run independently from the trigger:
+
+  - When using `LoopOnce` as the loop-option they will work fine on the first trigger but show only the last animation frame on consecutive triggers.
+  - When using `LoopForever` the animation will work fine on the first trigger but start and end on the wrong frame.
+
+
+The slightly annoying but in this case bearable workaround is animating by hand which means creating a Sprite for every frame of the animation and fade it in and out with the delay according to its position within the animation.
+If you understood how an animation works, this should be trivial to do. Refer to the tutorial on :term:`animation` if you have trouble.
+
+.. warning:: So we learned that you can't use triggers on animations if they are supposed to be displayed more than once.
+  For recreating the animation with triggers on its individual sprites you have to explicitly fade them out at the start of the trigger as they will otherwise fade in as soon as the trigger starts executing.
+
+.. note:: It should be noted that all combos in the map you're storyboarding for have to be at least as long as the longest transition effect (in this case 2 seconds of getting up).
+  Otherwise the effects will overlap or not flow well into each other (assuming you counteracted the overlapping issue) and make it look very bad. 2 seconds for a combo in osu! standard is very reasonable though so this would work in most beatmaps.
+
+.. You can check the result of this tutorial at <insert link here>.
+
+
+Trigger Groups
+--------------
```

Comment:
if you aren't planning to write these sections for this PR, you should probably add some todo blocks down here so they aren't forgotten. same with the headers at the top